src/ctia/flows/hooks.clj

```diff
@@ -24,7 +25,7 @@
 (defn reset-hooks! []
   (reset! hooks
           (-> empty-hooks
-              event-hooks/register-hooks)))
+              (event-hooks/register-hooks p/get-in-global-properties))))
```

Comment:
This is a convenient place to stop adding args and just reference the global properties service. Hooks becomes its own TK service which uses something like `reset-hooks!` in its `start` method.